docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastForest.tt

```diff
@@ -0,0 +1,23 @@
+﻿<#@ include file="RegressionSamplesTemplate.ttinclude"#>
+
+<#+
+string ClassHeader = @"// This example requires installation of additional NuGet package
+    // <a href=""https://www.nuget.org/packages/Microsoft.ML.FastTree/"">Microsoft.ML.FastTree</a>. ";
+
+string ClassName="FastForest";
+string Trainer = @"FastForest(labelColumnName: nameof(DataPoint.Label), featureColumnName: nameof(DataPoint.Features))";
+string TrainerOptions = null;
+
+string ExpectedOutputPerInstance= @"// Expected output:
+            // Label: 0.985, Prediction: 0.864
+            // Label: 0.155, Prediction: 0.164
+            // Label: 0.515, Prediction: 0.470
+            // Label: 0.566, Prediction: 0.501
+            // Label: 0.096, Prediction: 0.138";
```

Comment:
I think @rogancarr tried to make sure we have some distinction between real output and just comment, and he put TWO (2) spaces for actual output.
Would be nice you can keep that pattern.
EVERYWHERE